README.md

```diff
@@ -8,23 +8,6 @@ output:
 runtime: shiny
 ---
 
-Inputs {.sidebar}
```

Comment:
I'm a little confused on how to interpret these file changes. The README doesn't appear to be a [README in a traditional sense](https://www.mygreatlearning.com/blog/readme-file/). 

Are these changes supposed to be in the `Alumni` file? If so, should that file have the `.Rmd` suffix? Last I knew the file you were working from was an R Markdown. 


I would suggest putting instructions in the README about how to run the code that is in the `Alumni.Rmd` file. 
Aka, is it meant to be run from RStudio? Then click Run in RStudio? README's are super great when they give people the starting point of how to use your code. 

Let me know if this helps and/or makes sense.